content/blog/projects/current/zebulon.md

```diff
@@ -14,14 +14,20 @@ head:
       content: 2023-03-21T21:26:00Z
 ---
 
+
+<p>
+        <img src="https://github.com/Grallistrix/integra.content_RzecznikPatches/assets/116823604/445f7e75-925e-4744-995f-e778741361c4" alt="Image description" style="max-height: 500px;  display: block; margin: 0;">
```

Comment:
Use correct Markdown syntax and change the lastmod date.
If exact sizes are required, request a new component in the `integra.front` repo